generated/nifake/session.py

```diff
@@ -1074,7 +1075,8 @@ def write_waveform_numpy(self, waveform):
         Writes waveform to the driver
 
         Args:
-            waveform (list of float): Waveform data.
+            number_of_samples (int): How many samples the waveform contains.
+            waveform (numpy array of float64): Waveform data.
```

Comment:
Maybe we can append something to this to clarify that the fetched data is copied onto this argument.